backend/.env.example

```
@@ -5,3 +5,4 @@ SUPABASE_URL=""
 SUPABASE_SERVICE_ROLE_KEY=""
 # must be a securely generated random secret; keep it server-only and out of git
 ACTOR_KEY_SECRET="REPLACE_WITH_SECURE_RANDOM_SECRET"
+TRUST_RAILWAY_PROXY_HEADERS="false"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)